### ISSUE TITLE
Suggestion - fix grammar for time-based decisions

### ISSUE THREAD

**Issue by maxholloway** [protofire/solhint#353]
Hi! Tiny nit-pick here, nbd if it doesn't get fixed.

When including timestamps in contracts, you get the following warning.
```bash
warning  Avoid to make time-based decisions in your business logic  not-rely-on-time
```

Would we fix the grammar to say the following?
```bash
warning  Avoid making time-based decisions in your business logic  not-rely-on-time
```

Again, not a big deal, but it should be a super straightforward change in these files: 
* [docs/rules/security/not-rely-on-time.md](https://github.com/protofire/solhint/blob/991b8755ef2babcc2c97c278b039c86abeef4b8a/docs/rules/security/not-rely-on-time.md),
* [lib/rules/security/not-rely-on-time.js](https://github.com/protofire/solhint/blob/991b8755ef2babcc2c97c278b039c86abeef4b8a/lib/rules/security/not-rely-on-time.js), and 
* [docs/rules.md](https://github.com/protofire/solhint/blob/991b8755ef2babcc2c97c278b039c86abeef4b8a/docs/rules.md).

Otherwise, I've had a great experience using the tool! Thanks folks.

**Comment by MCarlomagno** [comment]
I just created a PR for this one

**Comment by dbale-altoros** [comment]
merged
